airflow/hooks/hive_hooks.py

```diff
@@ -775,6 +801,11 @@ def get_results(self, hql, schema='default', arraysize=1000):
                 'header': [],
             }
             cur = conn.cursor()
+            env_context = get_context_from_env_var()
```

Comment:
This could possible be a lot of variables, right?

Reply:
It can be. It is bounded by the # of entries in AIRFLOW_VAR_NAME_FORMAT_MAPPING, which is 4 right now.